ide/libs.bytelist/nbproject/project.xml

```diff
@@ -24,7 +24,15 @@
     <configuration>
         <data xmlns="http://www.netbeans.org/ns/nb-module-project/2">
             <code-name-base>org.netbeans.libs.bytelist</code-name-base>
-            <module-dependencies/>
+            <module-dependencies>
+                <dependency>
+                    <code-name-base>org.netbeans.libs.jcodings</code-name-base>
```

Comment:
Extracting this library shared between two modules is a good move. Please also remove a reference to it from `nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps`.

Reply:
[Done](https://github.com/apache/netbeans/pull/2812#discussion_r596641388).